logic/bitcoind.js

```diff
@@ -70,13 +70,12 @@ async function getLocalSyncInfo() {
   var blockChainInfo = info.result;
   var chain = blockChainInfo.chain;
   var blockCount = blockChainInfo.blocks;
-  var headerCount = blockChainInfo.headers;
-
-  const percentSynced = (Math.trunc(blockCount / headerCount * 10000) / 10000).toFixed(4); // eslint-disable-line no-magic-numbers, max-len
+  var headerCount = blockChainInfo.headers; 
+  var percent = blockChainInfo.verificationprogress;
 
   return {
-    chain: chain,
-    percent: percentSynced,
+    chain,
```

Comment:
@EddWills95  Aren't you changing the API with this? I'm not completely sure, but to me it seems like the property chain would now be called 0

Reply:
He doesn't change the returned object here, what is the issue?

Reply:
Sorry, misunderstood something